cpp/src/plasma/common.h

```diff
@@ -41,6 +41,23 @@ namespace plasma {
 
 enum class ObjectLocation : int32_t { Local, Remote, Nonexistent };
 
+enum class PlasmaErrorCode : int8_t {
+  PlasmaObjectExists = 1,
+  PlasmaObjectNonexistent = 2,
+  PlasmaStoreFull = 3,
+  PlasmaObjectAlreadySealed = 4,
+};
+
+ARROW_EXPORT arrow::Status MakePlasmaError(std::string message, PlasmaErrorCode code);
```

Comment:
Would probably be less confusing to follow the `Status(code, message)` convention rather than reverting it.

Reply:
Fixing it.